webapp/src/components/StatusCheck.tsx

```diff
@@ -1,14 +1,44 @@
-import { Box, Typography } from "@mui/material";
+import { Box, capitalize, Stack, Typography } from "@mui/material";
 import noData from "assets/launch.svg";
 import React from "react";
 import { useParams } from "react-router-dom";
 import { getStatusEndpoint } from "services/api";
 import Loading from "./Loading";
+import Grid from "@mui/material/Grid";
+import Paper from "@mui/material/Paper";
+import CheckIcon from "./Icons/Check";
+import ErrorIcon from "@mui/icons-material/Error";
+import PendingOutlinedIcon from "@mui/icons-material/PendingOutlined";
 
 type Props = {
   children: React.ReactNode;
 };
 
+interface StringByString {
+  [key: string]: any;
+}
+
+const STATUS_ICON_MAPPING: StringByString = {
+  finished: <CheckIcon />,
+  not_started: <CheckIcon />,
+  pending: <PendingOutlinedIcon />,
+  error: <ErrorIcon />,
+  lost: <ErrorIcon />,
```

Comment:
We can give icons a color. For the check, `color="success"`, and here:
```suggestion
  error: <ErrorIcon color="error" />,
  lost: <ErrorIcon color="error" />,
```